package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mia-platform-doc",
-  "version": "11.4.5",
+  "version": "11.4.6",
```

Comment:
why this version change?

Reply:
Because it is necessary to release it into production. 
I used the same procedure as I did the previous time